# monoStream.msg
int64 mono_timestamp
float64 x_mono
float64 y_mono

